hexrd/ui/resources/ui/main_window.ui

```diff
@@ -496,6 +497,14 @@
     <string>Current View</string>
    </property>
   </action>
+  <action name="action_export_to_maud">
+   <property name="enabled">
+    <bool>false</bool>
+   </property>
+   <property name="text">
+    <string>To Maud</string>
+   </property>
+  </action>
```

Comment:
Can we add a tooltip like the following?

```
Export to Maud ESG format. Only available in the polar view.
```

Reply:
Added in, thanks!